proxy-node-gateway/src/test/java/uk/gov/ida/notification/apprule/HubResponseAppRuleTests.java

```diff
@@ -155,7 +156,7 @@ public void redisCanStoreCertificateInSession() throws Throwable {
         assertThat(response.getStatus()).isEqualTo(200);
 
         final List<HubResponseTranslatorRequest> translatorArgs = TEST_TRANSLATOR_RESOURCE.getTranslatorArgs();
-        assertThat(translatorArgs.get(0).getConnectorEncryptionCertificate()).isEqualTo(STUB_COUNTRY_PUBLIC_PRIMARY_CERT);
+        assertThat(translatorArgs.get(0).getEidasIssuerEntityId()).isEqualTo(URI.create(SAMPLE_ENTITY_ID));
```

Comment:
Should the name of the test be changed as well?

Reply:
yep